tour6-deploying-with-promotions_5bb91da12c7d3a04dd5b5d8f.md

```diff
@@ -44,10 +51,10 @@ blocks:
             - echo 'Deploying to production!'
 </code></pre>
 
-In the Semaphore web interface, users will see a "Production Deploy" button
-once all blocks in the pipeline defined in `semaphore.yml` pass.
-When a user promotes a revision, Semaphore records the time and name of the
-person who initiated it, and proceeds by executing the pipeline defined in
+In the Semaphore 2.0 web interface, users will see a `Production Deploy` button
```

Comment:
Let's not put that in code block please. It appears in UI, not code or terminal output.

Reply:
Done.